docs/sessions/2026-05-23-upstream-proxy-hardening-work-it.md

```
@@ -0,0 +1,156 @@
+date: 2026-05-23 07:59:50 EST
+repo: git@github.com:jmagar/lab.git
+branch: fix/upstream-proxy-hardening
+head: 3919c74a
+plan: docs/superpowers/plans/2026-05-23-upstream-proxy-hardening.md
+agent: Claude (Opus 4.7)
+working directory: /home/jmagar/workspace/lab/.worktrees/upstream-proxy-hardening
+worktree: /home/jmagar/workspace/lab/.worktrees/upstream-proxy-hardening
+pr: #69 — fix(upstream): harden MCP proxy — process orphan, body OOM, UTF-8 panic — https://github.com/jmagar/lab/pull/69
+---
+
+## User Request
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)